pkg/sfu/downtrack.go

```diff
@@ -995,3 +1045,26 @@ func (d *DownTrack) DebugInfo() map[string]interface{} {
 		"Stats":               stats,
 	}
 }
+
+func (d *DownTrack) GetConnectionScore() float64 {
+	d.statsLock.Lock()
+	defer d.statsLock.Unlock()
+	return d.connectionStats.Score
+}
+
+func (d *DownTrack) updateStats() {
+	for {
+		select {
+		case _, ok := <-d.done:
+			if !ok {
```

Comment:
I think this could be simplified to `case <- d.done:`. we are not writing into done so the only input for it is close.